lib/simulator-xcode-6.js

```diff
@@ -772,6 +772,19 @@ class SimulatorXcode6 extends EventEmitter {
     await settings.setReduceMotion(this, reduceMotion);
   }
 
+  /**
+   * Enable/Disable reduce transparency.
+   *
+   * @param {boolean} reduceMotion - Whether or not to enable it.
+   */
+  async setReduceTransparency (reduceTransparency = true) {
+    if (await this.isFresh()) {
+      await this.launchAndQuit(false, STARTUP_TIMEOUT);
```

Comment:
I would say it is an unexpected side effect to have this call in the setter. I would rather remove it (as well as from `setReduceMotion`) and let the caller to decide

Reply:
Sure, I'll remove it.